tests/conftest.py

```diff
@@ -42,6 +42,16 @@
 from tests.setup_nodes import setup_farmer_multi_harvester
 
 
+async def wallet_is_synced(wallet_node: WalletNode, full_node_api) -> bool:
+    wallet_height = await wallet_node.wallet_state_manager.blockchain.get_finished_sync_up_to()
+    full_node_height = full_node_api.full_node.blockchain.get_peak_height()
+    return wallet_height == full_node_height
+
+
+async def wallets_are_synced(wns: List[WalletNode], full_node_api) -> bool:
+    return all([await wallet_is_synced(wn, full_node_api) for wn in wns])
```

Comment:
```suggestion
    return all(await wallet_is_synced(wn, full_node_api) for wn in wns)
```

Reply:
May as well change this since it's trivial and a button click away from being committed.

Reply:
Doesn't work for me, `E   TypeError: 'async_generator' object is not iterable`.

I guess all doesn't work with async iters

Reply:
I played briefly locally and don't have a solution so we can move on for now.  I'm _pretty_ sure that's not an async generator though...  that'd involve `async for` afaik.